doc/nrf/doc_styleguide.rst

```diff
@@ -16,15 +12,17 @@ NCS documentation is written in two formats:
 RST style guide
 ***************
 
-See :ref:`zephyr:doc_guidelines` for a short introduction to RST and, most importantly, to the conventions used in Zephyr.
+See Zephyr's :ref:`zephyr:doc_guidelines` for a short introduction to RST and, most importantly, to the conventions used in Zephyr.
 More information about RST is available in the `reStructuredText Primer`_.
 
 The |NCS| documentation follows the Zephyr style guide, and adds a few more restrictive rules:
 
 * Headings use sentence case, which means that the first word is capitalized, and the following words use normal capitalization.
 * Do not use consecutive headings without intervening text.
-* For readability reasons, start every sentence on a new line in the source files.
-  In the output, consecutive lines without blank lines in between are combined into one paragraph.
+* For readability reasons, every sentence should start on a new line in the source files and do not add line breaks within the sentence.
+In the output, consecutive lines without blank lines in between are combined into one paragraph.
+* For the conceptual documentation written in RST, you can have more than 80 characters per line.
```

Comment:
I don't see that as a separate point (more a note to the previous), but up to you how you want to do it. Just mentioning in case you missed it in my earlier comment. ;-)